homework/yaryna/decorator_hw_9.py

```diff
@@ -0,0 +1,22 @@
+name = input("Enter decorator name: ")
+text = input("Enter text for function: ")
+
+def decorator(foo,name = name):
+    def wrapper(text):
+        return (f"<{name.lower()}>" f"{foo(text)}" f"</{name.lower()}>")
+    return wrapper
+
+if name.isalpha() is True:
```

Comment:
popular beginner's mistake is when you tries to compare value which method or function returns with with boolean value. So you do not need to check if True is True, because method returns boolean object.